src/janitor.c

```diff
@@ -743,5 +744,10 @@ void config_postprocess(void)
 			db->pool_size = cf_default_pool_size;
 		if (db->res_pool_size < 0)
 			db->res_pool_size = cf_res_pool_size;
+		if (db->forced_user != NULL) {
+			// Initialize pool object for each database that has a forced user.
+			// Only creates data structures. Does not create connections.
```

Comment:
Does this mean that `min_pool_size_requires_connections` only works for forced users?

Reply:
👋 Would love to resurrect this PR and get it over the finish line.

I'm unclear about this change. Does this _need_ to happen for `db`s with forced users? Does this need to happen for  all `db`s? Could I get some guidance, please?